DashboardClient/OpcUaTypeReader.cpp

```diff
@@ -56,6 +56,8 @@
                         std::shared_ptr<ModelOpcUa::StructureBiNode>>>();
             initialize(notFoundObjectTypeNamespaces);
             LOG(INFO) << "Browsing variable types.";
+            this->BaseDataVariableTypes.push_back(NodeId_BaseVariableType);
+            this->BaseDataVariableTypes.push_back(ModelOpcUa::NodeId_t({"", "i=63"}));
```

Comment:
These two lines seems to be redundant.

Reply:
They are not redundant, the first one pushes back wit the full Namespace the second one without namespace. On the type reading procedure I needed this for some reason, otherwise some BaseVariablesTypes were not recognized correctly as BaseVariableTypes.

Reply:
Is the one with the full namespace not required at all? Wouldn't it be easier to ensure that all ns0 nodes are identical?